config/fcos/v1_5_exp/translate_test.go

```diff
@@ -1417,3 +1417,59 @@ func TestTranslateBootDevice(t *testing.T) {
 		assert.NoError(t, translations.DebugVerifyCoverage(actual), "#%d: incomplete TranslationSet coverage", i)
 	}
 }
+
+// TestTranslateExtensions tests translating the Butane config extensions section.
+func TestTranslateExtensions(t *testing.T) {
+	tests := []struct {
+		in         Config
+		out        types.Config
+		exceptions []translate.Translation
+		report     report.Report
+	}{
+		// empty config
```

Comment:
This is not an empty config.  :slightly_smiling_face:

Reply:
Wooops. Left behind from copy/paste from the original test. 😀 addressing asap